File: Python/card.py
from payment import Payment

class Card(Payment):
    cardNumber = int 
    cvv = int
    cardDate = str

    def Card(self, id, cardNumber, cvv, cardDate):
        super.__init__(id)
        self.cardNumber = cardNumber
        self.cvv = cvv
        self.cardDate = cardDate

File: Java/Card.java
class Card extends Payment{
    Integer numberCard;
    String dateCard;
    Integer cvv;

    public Card(Integer id, Integer numberCard, String dateCard, Integer cvv){
        super(id);
        this.numberCard = numberCard;
        this.dateCard = dateCard;
        this.cvv = cvv;
    }
}
